src/__test__/componentTests/App.test.jsx

```diff
@@ -0,0 +1,20 @@
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import { MemoryRouter } from 'react-router-dom';
+import App from '../../components/App';
+import store from '../../redux/configureStore';
+
+describe('App tests', () => {
+  it('it renders', () => {
+    const app = renderer
+      .create(
+        <MemoryRouter>
+          <Provider store={store}>
+            <App />
+          </Provider>
+        </MemoryRouter>,
+      )
+      .toJSON();
+    expect(app).toMatchSnapshot();
+  });
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.), Please add unit tests for pure functions in your Redux code, not only snapshot tests as you did.